cli/src/clojure_lsp/server.clj

```diff
@@ -488,8 +482,7 @@
     (->> edit
          (coercer/conform-or-log ::coercer/workspace-edit-or-error)
          ApplyWorkspaceEditParams.
-         (.applyEdit client)
-         .get))
```

Comment:
Removing this `.get` was required to get the integration tests passing. Otherwise, they freeze, presumably waiting for this CompletableFuture to resolve, which it never does. This change doesn't seem to break anything because nothing depends on the return value of publish-workspace-edit. I didn't look into how the `client` works in the tests, nor whether there is a way to make it resolve the Future.

Reply:
I think this get was there to wait for this action to finish, but I think there is no usage waiting for that, so it's probably ok